core/chains/evm/txmgr/eth_confirmer.go

```diff
@@ -313,16 +311,9 @@ AND eth_txes.id = eth_tx_attempts.eth_tx_id AND eth_txes.evm_chain_id = $2`,
 //
 // This scenario might sound unlikely but has been observed to happen multiple times in the wild on Polygon.
 func (ec *EthConfirmer) CheckConfirmedMissingReceipt(ctx context.Context) (err error) {
-	var attempts []EthTxAttempt
-	err = ec.q.Select(&attempts,
-		`SELECT DISTINCT ON (eth_tx_attempts.eth_tx_id) eth_tx_attempts.*
-		FROM eth_tx_attempts
-		JOIN eth_txes ON eth_txes.id = eth_tx_attempts.eth_tx_id AND eth_txes.state = 'confirmed_missing_receipt'
-		WHERE evm_chain_id = $1
-		ORDER BY eth_tx_attempts.eth_tx_id ASC, eth_tx_attempts.gas_price DESC, eth_tx_attempts.gas_tip_cap DESC`,
-		ec.chainID.String())
+	attempts, err := ec.orm.FindEtxAttemptsConfirmedMissingReceipt(ec.chainID)
 	if err != nil {
-		return errors.Wrap(err, "CheckConfirmedMissingReceipt failed to query")
```

Comment:
felt it was more appropriate to return the orm-specific error

Reply:
nit: still might be nice to still have the error wrapped with the func that called the ORM func - it can make it easier to trace error logs

Reply:
It ends up being rewrapped into an error from the parent function so the trace is still the same.